tests/test_solvermode/test_materials.py

```diff
@@ -6,9 +6,11 @@
 @pytest.mark.setup
 def test_solver_material(load_mixing_elbow_mesh):
     session = load_mixing_elbow_mesh
-    session.solver.root.setup.materials.copy_database_material_by_name(
+    # session.solver.root.setup.materials.copy_database_material_by_name(type="fluid", name="water-liquid")
+    session.solver.root.setup.materials.database.copy_by_name(
         type="fluid", name="water-liquid"
     )
```

Comment:
Tests are run for both backend versions and `database` attribute only exists since 2023R1, so this fails for 2022R2.

```suggestion
    # session.solver.root.setup.materials.copy_database_material_by_name(type="fluid", name="water-liquid")
    try:
        session.solver.root.setup.materials.database.copy_by_name(
            type="fluid", name="water-liquid"
        )
    except AttributeError:
        session.solver.root.setup.materials.copy_database_material_by_name(
            type="fluid", name="water-liquid"
        )
```

Reply:
@akulshre-qa is it currently possible to configure a test such that it's only enabled for a specific backend? Can we query the server version at runtime through pyfluent?